mavlink/test_flight_input.py

```diff
@@ -0,0 +1,22 @@
+"""
+Test for flight input device by printing to console.
+"""
+
+from modules import flight_input_device
+from time import sleep
```

Comment:
System modules are at top without empty line between docstring and import. 1 line between system imports and non system imports.